### ISSUE TITLE
Epoch x axis value is off by one

### ISSUE THREAD

**Issue by jondo** [tensorflow/tensorboard#5805]
## Environment information (required)

TensorBoard 2.9.1

## Issue description

`tensorflow.keras.Model.fit` prints epoch numbers, starting with 1. In TensorBoard, however, the displayed log curves start at a Step value of 0. I would expect them to also start at 1.

**Comment by bileschi** [comment]
When using the TensorBoard summary writer, the step value will default to `None`, which will externalize control.  This external control mechanism within TensorBoard begins at 0.  I suspect what is happening here is that Keras writes out it's first value *after* the first epoch, but does not pass an explicit step value.   It may be possible to update this by changing Keras to make the log & summary write.  However, from TensorBoard's perspective this is WAI.

https://www.tensorflow.org/api_docs/python/tf/summary/SummaryWriter

**Comment by jondo** [comment]
Thank you for the feedback! I have now [asked the Keras community about this](https://discuss.tensorflow.org/t/syncing-the-epoch-numbers-of-model-fit-and-tensorboard/11068).
